presto-spi/src/main/java/com/facebook/presto/spi/block/DictionaryBlock.java

```diff
@@ -255,7 +256,7 @@ public void retainedBytesForEachPart(BiConsumer<Object, Long> consumer)
     {
         consumer.accept(dictionary, dictionary.getRetainedSizeInBytes());
         consumer.accept(ids, sizeOf(ids));
-        consumer.accept(this, (long) INSTANCE_SIZE);
+        consumer.accept(this, (long) INSTANCE_SIZE + DICTIONARY_ID_SIZE);
```

Comment:
@nezihyigitbasi : Agree. I "merge" them because I feel it's a bit waste to register it separately. But probably that's OK :)